apps/api/src/database/scripts/seed-service.ts

```diff
@@ -119,7 +119,7 @@ export class SeedService {
             action = cleanText(action);
             if (action.includes('y')) {
               allowedAction = Permissions.PERFORM;
-            } else if (action.includes('l')) {
+            } else if (action.includes('lc')) {
```

Comment:
Nit - In future, for readability it might be good to replace this with a constant.

Reply:
it is a seed service, explicitly done to cater to client mistakes.